easy_access/db/update.py

```
@@ -550,7 +553,22 @@ async def execute_bulk_database_operations(
             )
-            await CopyrightItem.bulk_update(change_batch, fields=changed_fields)
+            # Convert Tortoise bulk_update to SQLAlchemy bulk update
+            async for session in get_session():
+                for item in change_batch:
+                    # Update each item individually since SQLAlchemy doesn't have direct bulk_update equivalent
+                    update_data = {
+                        field: getattr(item, field)
+                        for field in changed_fields
+                        if hasattr(item, field)
+                    }
+                    update_data["modified_at"] = datetime.now()
```

Ruff (0.14.6)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)